geonode/people/tests.py

```diff
@@ -911,3 +912,116 @@ def test_delete_a_user_with_resource(self):
         # bobby cant be deleted
         self.assertNotEqual(get_user_model().objects.filter(username="bobby").first(), None)
         self.assertTrue("user_has_resources" in response.json()["errors"][0])
+
+    def test_remove_self_from_group_manager_all(self):
+        """
+        user is manager of some groups
+        and is quitting all
+        """
+        bobby = get_user_model().objects.get(username="bobby")
+        for group in self.group_profiles:
+            group.join(bobby)
+            group.promote(bobby)
+
+        self.assertTrue(self.client.login(username="bobby", password="bob"))
+        self.assertTrue(bobby.is_authenticated)
+        # assert that bobby is manager of these groups
+        for group in self.group_profiles:
+            self.assertTrue(bobby in group.get_managers())
+        # call the api that removes him from all assigned groups as manager
+        response = self.client.post(
+            path=f"{reverse('users-list')}/{bobby.pk}/remove_from_group_manager",
+            data={"groups": "ALL"},
+        )
+        self.assertTrue(response.status_code == 200)
+        # check that bobby is manager no more
+        for group in self.group_profiles:
+            self.assertFalse(bobby in group.get_managers())
+            self.assertTrue(group.title in response.json()["success"])
+
+    def test_remove_self_as_group_manager_list(self):
+        """
+        user is manager of some groups
+        and is quitting some of them
+        """
+        bobby = get_user_model().objects.get(username="bobby")
+
+        for group in self.group_profiles:
+            group.join(bobby)
+            group.promote(bobby)
+
+        self.assertTrue(self.client.login(username="bobby", password="bob"))
+        self.assertTrue(bobby.is_authenticated)
+        # assert that bobby is manager of these groups
+        for group in self.group_profiles:
+            self.assertTrue(bobby in group.get_managers())
+        # call the api that removes him from all assigned groups as manager
+        response = self.client.post(
+            path=f"{reverse('users-list')}/{bobby.pk}/remove_from_group_manager",
+            data={"groups": ",".join([str(group.group_id) for group in self.group_profiles[:3]])},
```

Comment:
isnt better to use this as a list instead of string?  Otherwise in the API you have to re-generate the list